python/fledge/plugins/south/mqtt-readings/mqtt-readings.py

```diff
@@ -242,7 +249,7 @@ def plugin_register_ingest(handle, callback, ingest_ref):
 class MqttSubscriberClient(object):
     """ mqtt listener class"""
 
-    __slots__ = ['mqtt_client', 'broker_host', 'broker_port', 'username', 'password', 'topic', 'qos', 'keep_alive_interval', 'asset', 'loop']
+    __slots__ = ['mqtt_client', 'broker_host', 'broker_port', 'username', 'password', 'topic', 'qos', 'keep_alive_interval', 'json_object_key', 'asset', 'loop']
```

Comment:
see my comment above, We can name it a self explanatory long variable name same as config item key i.e. reading_datapoint_name_for_primitive_value